lib/sales_engine.rb

```diff
@@ -4,7 +4,7 @@
 # require_relative './invoice_item_repository'
 # require_relative './item_repository'
 # require_relative './merchant_repository'
-# require_relative './transaction_repository'
+require_relative './transaction_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.